apps/frontend/src/libs/modules/notifications/libs/enums/notification-option-values.enum.ts

```diff
@@ -0,0 +1,5 @@
+const NotificationOptionValues = {
```

Comment:
Seems that NotificationOptionValues is not an enum actually, so there is no need to define an object. Move AUTO_CLOSE_TIME out just like a common constant value

Reply:
Also move it to a constant file